extensions/zalgo-text/package.json

```
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "zalgo-text",
+  "title": "Zalgo Text Generator",
+  "description": "Translates given text to Zalgo text",
+  "icon": "command-icon.png",
+  "author": "petermiles",
+  "categories": [
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "G̙͖̬̳̜͛̓͘e̶̟ͧ͗n̷̳̤̉ͩ̓͗̈́̇ͪ͡ē̵̫͓̳̹̊ṟ̷ͩ͋a̱̟͍͉̼͖ͤͫͥ̒̿́t̪̥̼͍͖͓̳͒ͫ͂̐ͪ̿e̒̒͏̬̰̠̖̝̘̯ͅ",
+      "description": "Convert text to Z̬͚̭̤͓̟͖͐͒ȃ̮̙̯͈͍͇͒ͤ̓ͤ́ͨͅͅl̷̐̍̃̊͏̝ğ̡̪̪͓͈͙̘̽͐͒̆́ö̮̳̪̯̟̩̭̎ͮ̇̿̀͢ ̥͎̂̿͋͋́̀̕͢T̴̷̳̟̟͎͛͌̔̈̇̓ę̠̊͑͊̉̽̓̑̒x̸̠̭͎͍̋̏̇ͯ͋̍ͣt̷̶̤͇̠̜̩̯̦͑̈́ͤ̿͛ͤ̀",
```

```suggestion
      "description": "Convert text to Zalgo",
```
You could also add an [argument](https://developers.raycast.com/information/lifecycle/arguments)